src/map/ai/ai_mob_dummy.cpp

```diff
@@ -719,6 +719,7 @@ void CAIMobDummy::ActionAbilityStart()
     // We don't have any skills we can use, so let's go back to attacking
     if (MobSkills.size() == 0)
     {
+        m_PMob->PBattleAI->SetMobAbilityEnabled(false);
         TransitionBack(true);

```

Comment:
If you meant mobs starts with no skills then skill get unlocked during the fight..please show me any one mob that A) actually does that and B) is not capable of setting that bool again when it does so.